File: HelpSoftware/Help/EF/ArticleComment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Help
{
  public class ArticleComment
  {
    [Key]
    public int ID_ArticleComment { get; set; }

    public string Comment { get; set; }

    public string CommentTitle { get; set; }

    public User UserComment { get; set; }

    public Article CommentArticle { get; set; }

    public DateTime CommentTime { get; set; }    
  }
}

File: HelpSoftware/Help/Help.Web/Web/src/app/EF/ArticleComment.ts
import { User, Article } from "./EF";

export class ArticleComment {
  ID_ArticleComment: Number;
  Comment: string;
  CommentTitle: string;
  UserComment: User;
  CommentArticle: Article;
  CommentTime: Date;

  deserialize(input) {
    this.ID_ArticleComment = input.ID_ArticleComment;
    this.Comment = input.Comment;
    this.CommentTitle = input.CommentTitle;
    this.UserComment = new User().deserialize(input.UserComment);
    this.CommentArticle = input.CommentArticle;
    this.CommentTime = input.CommentTime;
    return this;
  }
}
